org.libvips.nip4.json

```diff
@@ -0,0 +1,478 @@
+{
+    "app-id" : "org.libvips.nip4",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nip4",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=home",
+        "--filesystem=/mnt",
+        "--filesystem=/srv"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/libheif",
+        "/lib/x86_64-linux-gnu",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/lib/openjpeg-*",
+        "/man",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/aclocal*",
+        "/share/autoconf*",
+        "/share/automake*",
+        "/share/info",
+        "/share/thumbnailers",
+        "/share/USING_HDF5_CMake.txt",
+        "/share/COPYING",
+        "/share/RELEASE.txt",
+        "/share/cmake",
+        "/bin/a*",
+        "/bin/b*",
+        "/bin/c*",
+        "/bin/d*",
+        "/bin/e*",
+        "/bin/f*",
+        "/bin/g*",
+        "/bin/h*",
+        "/bin/i*",
+        "/bin/l*",
+        "/bin/m*",
+        "/bin/o*",
+        "/bin/p*",
+        "/bin/r*",
+        "/bin/s*",
+        "/bin/t*",
+        "/bin/x*",
+        "/bin/vips",
+        "/bin/vipsheader",
+        "/bin/vipsedit",
+        "/bin/vipsprofile",
+        "/bin/vipsthumbnail",
+        "/bin/jxlinfo",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+
+        {
+            "name" : "highway",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "build-options" : {
+                "config-opts" : [
+                    "-DCMAKE_INSTALL_LIBDIR=lib",
```

Comment:
you shouldn't need to set LIBDIR. make sure to have flatpak-builder 1.4.4 or the flatpak version

Reply:
I switched to builder 1.4.4 and indeed the `lib=` seems to be set by default now, thanks